app/models/user.rb

```diff
@@ -34,6 +34,8 @@ class User < ActiveRecord::Base
 
   belongs_to :system_role, class_name: "Role"
 
+  DEFAULT_PROFILE_PIC_URL = 'http://coursemology.s3.amazonaws.com/public/default_profile_pic.png'
```

Comment:
:+1:

> @jsyeo I changed to use constants leh.

By the way, this is not the right place to use *leh*. :stuck_out_tongue: Just saying. heh.

![lol](https://31.media.tumblr.com/3dd339dcd3e49d3469b197ca9f0479c3/tumblr_inline_nfudqiobSq1t1tlzz.gif)